acipher/ta/user_ta_header_defines.h

```diff
@@ -15,12 +15,25 @@
 #define TA_UUID				TA_ACIPHER_UUID
 
 #define TA_FLAGS			TA_FLAG_EXEC_DDR
+
+/* Provisioned stack size */
 #define TA_STACK_SIZE			(2 * 1024)
+
+/* Provisioned heap size for TEE_Malloc() and friends */
 #define TA_DATA_SIZE			(32 * 1024)
 
+/* The gpd.ta.version property */
+#define TA_VERSION	"1.0"
+
+/* The gpd.ta.description property */
+#define TA_DESCRIPTION	"Example of TA using asymmetric cipher"
+
+/* Extra properties */
 #define TA_CURRENT_TA_EXT_PROPERTIES \
-    { "gp.ta.description", USER_TA_PROP_TYPE_STRING, \
-        "Example of TA using asymmetric cipher" }, \
-    { "gp.ta.version", USER_TA_PROP_TYPE_U32, &(const uint32_t){ 0x0010 } }
+    { "org.linaro.optee.examples.acipher.property1", \
```

Comment:
I don't think this should be here, the reason for seeing that in "hello world" is what Jerome wrote in the commit message in https://github.com/linaro-swg/optee_examples/commit/20e5a612e6303, i.e., 
```
To still illustrate
the mechanism used to define non-standard properties, we add a couple of
values with names that follow the recommended reverse domain name
convention. Other TAs are left with no additional properties and default
version and description for simplicity.
```
So, I believe you can skip the `TA_CURRENT_TA_EXT_PROPERTIES` changes done here in in the other `user_ta_header_defines.h` in this patch set. I.e., the `TA_VERSION` and `TA_DESCRIPTION` changes should be sufficient.